projects/common/src/index.ts

```diff
@@ -162,12 +162,23 @@ export interface AtomElement {
     css?: string[]
     js?: string[]
 }
+
+export interface MediaAtomElement {
+    id: 'media⚛︎'
+    image?: Image
+    html: string
+    platform?: 'youtube' | 'dailymotion' | 'mainstream' | 'url'
+    assetId?: string
+    title?: string
+}
+
 export type BlockElement =
     | HTMLElement
     | ImageElement
     | UnknownElement
     | TweetElement
     | AtomElement
```

Comment:
What do we need AtomElement for here now?

Reply:
All other kinds of atom which may be present. (And are being rendered by the atom renderer)